docs/learn/learn-bridges.md

```diff
@@ -64,8 +68,12 @@ chains, `main` and `side`. Ether deposited into the contract on `main` generates
 denominated in ERC-20 tokens on `side`. Conversely, ERC-20 tokens deposited back into the contract
 on `side` can free up Ether on `main`.
 
-> To learn more on how Bitcoin and Ethereum can Cooperate and Collaborate Through Polkadot, check
-> out this explainer video [here](https://www.youtube.com/watch?v=rvoFUiOR3cM)
+:::note Explainer video on Ethereum x Polkadot
```

Comment:
Title not needed for this one. Ethereum x Polkadot does not convey the summary of the video, in fact is misleading.
```suggestion
:::note 
```